### DISCUSSION TITLE
Vercel toolbar suddenly stopped showing for our deployments

### DISCUSSION THREAD

**Discussion by ingljoOpt** [vercel/community#5618] [Bug Report]
### Page to Investigate

https://optimal-test.vercel.app/

### Steps to Reproduce

In the build log it says:
[12:08:15.553] Injected preview comments middleware, you can disable this in your project settings.

But the toolbar is missing when user visit the preview deployment.

It was working until yesterday.

Might it be related to new release yesterday?
https://vercel.com/changelog/webhooks-are-now-generally-available

Comments are enabled on project settings for all preview builds.

**Comment by javierfuentesm** [comment]
Everything was working fine for my app and suddenly it stopped working I tried redeployed and nothing works, even though I didn't deploy anything new in 43 days

**Reply by tugrulates** [reply]
This is happening on my projects as well.

I just created a new website and a new project, and I have the same issue on it:
https://astro-test-n0dtbui9y-tugrulates.vercel.app

Happy to assist in debugging.

**Comment by eramdam** [comment]
Seeing the same thing on deployments from today. Nothing changed in the project's configuration 😕

**Comment by orv1s** [comment]
Same here, some of the previous Preview deploys still show toolbar correctly but not the recent ones.

**Comment by sebastianroming** [comment]
We've just rolled out a fix and the toolbar should now show up again after a new deployment.

**Reply by javierfuentesm** [reply]
@sebastianroming  I thought my issue was related but it seems like not, any idea https://github.com/orgs/vercel/discussions/5619 ?

**Reply by ingljoOpt** [reply]
Thank you! I can confirm that the toolbar is visible again now after a new deployment.
